google/cloud/pubsub/samples/samples.cc

```diff
@@ -129,7 +129,8 @@ void ExampleStatusOr(google::cloud::pubsub_admin::TopicAdminClient client,
     // The actual type of `topic` is
     // google::cloud::StatusOr<google::pubsub::v1::Topic>, but
     // we expect it'll most often be declared with auto like this.
-    for (auto& topic : client.ListTopics(project_id)) {
+    for (auto& topic :
```

Comment:
This is an unrelated change, but I noticed it when I ran the auto version of samples